src/test/java/seedu/address/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,7 @@
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_PERSON = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_PERSON = Index.fromOneBased(3);
+    public static final Index INDEX_FIRST_EVENT = Index.fromOneBased(1);
```

Comment:
should we just use 3 indexes and call them INDEX_FIRST, INDEX_SECOND, and INDEX_THIRD? so they can be used by both events and contacts test cases?